src/libutil/CMakeLists.txt

```
@@ -47,6 +47,7 @@ function (setup_oiio_util_library targetname)
 
     target_include_directories (${targetname}
             PUBLIC
+                $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/deps/dist/include>
```

```suggestion
                $<BUILD_INTERFACE:${OpenImageIO_LOCAL_DEPS_ROOT}/include>
```

It's possible to override the location with a cmake variable.